scripts/expected_api_mismatch.txt

```diff
@@ -20,3 +20,5 @@ Parameter type mismatch in BrowserContext.cookies(return=): documented as List[{
 Method not implemented: Error.name
 Method not implemented: Error.stack
 Method not implemented: Error.message
+# Temporary
```

Comment:
This is not temporary. Let's change to `# Devices are properly typed`.

Reply:
Waiting for review, if wanted to add types to docs upstream, but would change comment instead